app/controllers/projects/samples/attachments/deletions_controller.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+module Projects
+  module Samples
+    module Attachments
+      # Controller actions for Project Samples Attachments Concatenation
+      class DeletionsController < Projects::Samples::ApplicationController
+        respond_to :turbo_stream
+
+        def new
+          authorize! @project, to: :update_sample?
+          render turbo_stream: turbo_stream.update('sample_files_modal',
+                                                   partial: 'modal',
+                                                   locals: {
+                                                     open: true
+                                                   }), status: :ok
+        end
+
+        def destroy # rubocop:disable Metrics/AbcSize
+          authorize! @project, to: :update_sample?
+
+          atts_to_delete = get_attachments(deletion_params['attachment_ids'])
```

Comment:
Can we update these variable names to attachments_to_delete. Just for clarity. This will require updates to the turbo stream files below as well where the variable is used. This also applies to the `get_attachments` method below

Reply:
Fixed in 62fbc1ba10cbef28ca5e0f5bc9aee03f98487c49